.github/workflows/pr-linter.yml

```diff
@@ -27,20 +27,29 @@ permissions: read-all
 
 # Kill stale checks
 concurrency:
-  group: ${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 
 jobs:
-  pr-formatting:
-    name: Formatting
-    runs-on: ubuntu-22.04
+  pr-commits:
+    name: Check commits
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
       - name: Check commit messages
         run: python3 ./.github/automation/commit-msg-check.py "${{ github.event.pull_request.head.sha }}" "${{ github.event.pull_request.base.sha }}"
+
+  pr-clang-format:
+    name: Formatting
```

Comment:
```suggestion
    name: Clang-Format
```

Reply:
To avoid messing up required check we can add something along these lines:
```
  status:
    needs: "Commit messages"
    needs: "clang-format"
    needs: "Format tags consistency"
    runs-on: ubuntu-latest
    name: "Formatting"
    steps:
      - name: Print success
        run: echo Success
```